siriuspy/siriuspy/pwrsupply/csdev.py

```diff
@@ -1459,10 +1545,13 @@ def _get_ps_FAC_2S_DCDC_propty_database():
                                         'prec': 2, 'unit': 'C'},
         'RelativeHumidityIIBMod1-Mon': {'type': 'float', 'value': 0.0,
                                         'unit': '%'},
-        'IntlkIIBMod1-Mon': {'type': 'int', 'value': 0},
-        'IntlkIIBMod1Labels-Cte': {'type': 'string',
-                                   'count': len(_et.IIB_INTLCK_FAC_2S_DCDC),
-                                   'value': _et.IIB_INTLCK_FAC_2S_DCDC},
+        'IntlkIIBMod1-Mon': {'type': 'int', 'value': 0, 'unit': 'interlock'},
+        'IntlkIIBMod1Labels-Cte': {
+            'type': 'string',
+            'count': len(_et.IIB_INTLCK_FAC_2S_DCDC),
+            'value': _et.IIB_INTLCK_FAC_2S_DCDC,
+            'unit': 'interlock',
+            },
```

Comment:
I think it is not necessary to break the line here

Reply:
done!